5.5/examples/replica/mysql_replica.json

```diff
@@ -16,35 +16,35 @@
       "value": "mysql-master"
     },
     {
-      "name": "MYSQL_MASTER_USER",
+      "name": "MYSQL_REPLICATION_USER",
       "description": "The username used for master-slave replication",
-      "value": "master"
+      "value": "replication"
     },
     {
-      "name": "MYSQL_MASTER_PASSWORD",
-      "description": "The password for the MySQL master user",
+      "name": "MYSQL_REPLICATION_PASSWORD",
+      "description": "The password used for the MYSQL_REPLICATION_USER",
       "generate": "expression",
       "from": "[a-zA-Z0-9]{12}"
     },
     {
       "name": "MYSQL_USER",
-      "description": "The username that clients will use to connect to MySQL server",
+      "description": "The username that clients will use to connect to the server",
       "value": "user"
     },
     {
       "name": "MYSQL_PASSWORD",
-      "description": "The password for the MySQL master user",
+      "description": "The password for the MYSQL_USER",
       "generate": "expression",
       "from": "[a-zA-Z0-9]{12}"
     },
     {
       "name": "MYSQL_DATABASE",
-      "description": "The name of the database that will be created",
+      "description": "The name of the database that will be created and replicated",
       "value": "userdb"
     },
     {
       "name": "MYSQL_ROOT_PASSWORD",
-      "description": "The password for the MySQL adminitrator",
```

Comment:
There was a typo here. Since the user name is 'root', I think it is clearer to say "root user".
(it took me some time to figure out what was the username for the admin user -- root)